src/main/resources/reference.conf

```diff
@@ -14,6 +14,17 @@
 
 akka-persistence-jdbc {
 
+  # If set to true event deletion is performed as a soft, logical delete. Events are kept in the journal and are sill
+  # delivered in queries. Otherwise, a hard delete will be performed. 
+  # 
+  # Note that even when configured for hard deletes, the last event is kept as a logical delete. 
+  # This is necessary because we must keep track of the highest sequence number that was ever used on a 
+  # given persistent actor. However, this last event will be 'invisible' it won't be ever replay nor delivered on queries.
+  #  
+  # This property affects jdbc-journal.logicalDelete and jdbc-read-journal.includeLogicallyDeleted.
+  #
+  logicalDeletion.enable = true
```

Comment:
I don't think logical deletes are supported by Akka Persistence any more (since long time back) so I find all "logical" stuff here confusing. I believe the plugin is using it for keeping track of the highest sequence number if all events are deleted, but that is an implementation detail and such events should never be shown in query results.

Reply:
That's the conclusion we had as well. So, we want to move to hard deletes and preserve the last one as logically deleted event because of the sequence nr.

The problem is that we should keep it backwards compatible, even if the previous behaviour doesn't make much sense. At least for me, it doesn't make sense at all.  For more context see discussion #162.

What do you suggest instead:
1. should we make hard delete the default case and have the config to allow users to have the old behaviour?
2. or should we simply drop the concept of logical deletes? Deleted is deleted.

Reply:
I think it's a bug to return such a deleted event from queries.
Makes no sense to see one deleted event but not others.

Reply:
That was not the behaviour, the behaviour was to logically delete them and still deliver on the query. All of them. 

But yeah, maybe we should consider that all as a bug because it makes no sense at all.

Reply:
ok, I thought it was some special case for the last event (sorry I haven't looked much at code here)

I agree that all logical deletion stuff should be removed in next "breaking change" release.